analysis/report_templates/default.html

```diff
@@ -228,21 +233,47 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                             </div>
                         </div> <!-- row -->
 
+                        {% if detailed_coverage_report%}
+                        <div class="row">
+                            <div class="col s6 offset-s3">
+                                <h5 class="center-align">Ranking by rare region covered</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.rare_region_ranking_plot }}">
+                                    The table shows the coverage rank of each fuzzer
+                                    on the rare regions for this benchmark. The rare
+                                    region is defined as only being hit by less than
+                                    threshold(default=20%) of all fuzzers.
+                            </div>
+                        </div> <!-- row -->
+
+                        <div class="row">
+                            <div class="col s6 offset-s3">
+                                <h5 class="center-align">Correlation table</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.correlation_plot }}">
+                                    The correlation table is a square matrix where each row and column
+                                    represents a fuzzer, and each cell contains a number showing the
+                                    percentage of regions covered by the fuzzer of the column but not by
```

Comment:
In your demo I see 945 so I assume this is not a percentage. And I'm not even sure what it would be a percentage of.